app/components/post_header/post_header.js

```diff
@@ -117,9 +117,16 @@ export default class PostHeader extends PureComponent {
             fromAutoResponder,
             overrideUsername,
             theme,
+            renderReplies,
+            shouldRenderReplyButton,
+            commentedOnDisplayName,
+            commentCount,
+            isBot,
         } = this.props;
 
         const style = getStyleSheet(theme);
+        const showReply = shouldRenderReplyButton || (!commentedOnDisplayName && commentCount > 0 && renderReplies);
+        const displayNameStyle = [style.displayNameContainer, showReply && isBot ? style.displayNameContainerBotReplyWidth : null];
```

Comment:
I believe this also applies (the bot tag) when the message is from a webhook or an autoresponder not only when the post is by a bot user

Reply:
Got it, I should have updated that area also, on it!